src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -369,6 +369,7 @@ protected virtual ImmutableArray<NamedTypeSymbol> GetAllInterfaces()
         /// TypeSymbol.Interfaces as the source of edge data, which has had cycles and infinitely
         /// long dependency cycles removed. Consequently, it is possible (and we do) use the
         /// simplest version of Tarjan's topological sorting algorithm.
+        /// Returned interfaces are unique with regards to tuple names.
```

Comment:
>/// Returned interfaces are unique with regards to tuple names. [](start = 8, length = 63)

Is this comment accurate? I thought difference in tuple names was not considered significant.